include/dem/grid_motion.h

```diff
@@ -109,14 +111,49 @@ class GridMotion
   move_grid_translational(
     parallel::distributed::Triangulation<dim> &triangulation);
 
+  /**
+   * Carries out translational motion of the triangulation
+   *
+   * @param triangulation Triangulation
+   */
+  void
+  move_grid_due_particles_forces(
+    parallel::distributed::Triangulation<dim> &triangulation);
+
+  /**
+   * Update member information like boundary_forces
+   */
+  void
+  update_parameters_before_motion();
```

Comment:
Need a better explanation of the function content. I don't understand what the function does from the comments.

Reply:
Next commit